foodsaving/conversations/helpers.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+# load list of lists of alternative names of emojis
+# gathered here:
+# https://github.com/markdown-it/markdown-it-emoji/ (continue)
+#    blob/f1bc7217e9eb4d954c145faf316718e263778e34/lib/data/full.json
+with open('foodsaving/conversations/emoji_names.json') as json_data:
```

Comment:
Quite a list! Maybe you could use the library from here: https://github.com/facelessuser/pymdown-extensions/blob/master/pymdownx/twemoji_db.py ?
We already use it for rendering emojis in message notifications, so you could directly import it via `from pymdownx import twemoji_db`.

Unrelated thought: the frontend to have a list of all emoji names, which makes me think if we should switch to a reduced version: https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/light.json